.vscode/settings.json

```diff
@@ -18,7 +18,7 @@
     "editor.defaultFormatter": "ms-python.black-formatter"
   },
   "editor.tabSize": 4,
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
(not too familiar with vscode, but should this file be part of the repo at all?)

Reply:
My version of code (possibly the same for you) has issues with black formatting, so I need to run it manually.
formatOneSave will messup files.